packages/graph/migrations/postgres/migration/1656417312397_initial.ts

```diff
@@ -199,6 +199,86 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
     },
   );
 
+  pgm.createTable("closed_entity_types", {
+    closed_type_id: {
+      type: "UUID",
+      notNull: true,
+      primaryKey: true,
+    },
+    // Unsure if we want the closed schema right now.
+    // closedSchema: {
+    //   type: "JSONB",
+    //   notNull: true,
+    // },
```

Comment:
I think we should leave it commented out until we populate it. [reference slack thread](https://hashintel.slack.com/archives/C03F7V6DU9M/p1667322298518239?thread_ts=1667322288.750449&cid=C03F7V6DU9M) (_internal_)

Reply:
We definitely want the closed schema right? But the "closed schema" will be a little misleading for now (so we should definitely add warnings) as we won't be writing the logic to actually close the thing.

I think we said we want to copy in the entity type so that we can start writing structural queries as if there was a closed one here. (Mainly so we can use JSON Path to search inside the `allOf` for inheritance)